### ISSUE TITLE
A11y_AzureDataStudioAAE_PG1_Compute + Storage_ScreenReader: "Worker node" spin button is announced as required by the screen reader but no visual identification for required field is given on the screen

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15769]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#STP;#NVDA;#Jaws;#WCAG3.3.1;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "PG1" and open manage and tab to "Compute + Storage" page
5. Open NVDA
6. Tab to "Worker node" spin button and check if required field identification is given on the screen

### Actual:
"Worker node" spin button is announced as required by the screen reader but no visual identification for required field is given on the screen.

### Expected:
"Worker node" spin button is announced as required by the screen reader and also visual identification for required field should be given on the screen.

### User Impact:
Users will get confused if no visual identification is given on the screen.

https://user-images.githubusercontent.com/85863543/127597307-1a21bcd7-e24f-4c98-a847-95f6dd6342c7.mp4

**Comment by krrish2021** [comment]
@candiceye the issue can not be verified as the "pg1" control is not available in latest azure data studio @nasc17 Please reopen the bug.
![Screenshot (182)](https://user-images.githubusercontent.com/85863543/150963293-2e4b79f8-f3d7-40b2-ad7b-efd7d86ca7e6.png)

**Comment by candiceye** [comment]
@krrish2021 Closing this bug as I just checked the postgres dashboard's Compute + Storage tab and it does show the field as required with a visual asterisk:
![image](https://user-images.githubusercontent.com/22386690/154414613-4e4191b2-753d-4d56-8fa9-aa11ad8954e8.png)
